src/sentry_auth_saml2/forms.py

```diff
@@ -1,11 +1,16 @@
 from __future__ import absolute_import
 
+import django
 from django import forms
-from django.forms.utils import ErrorList
 from django.utils.encoding import force_text
 from django.utils.translation import ugettext_lazy as _
 from onelogin.saml2.idp_metadata_parser import OneLogin_Saml2_IdPMetadataParser
 
+if django.VERSION >= (1, 8):
+    from django.forms.utils import ErrorList
```

Comment:
Why not just do like:

```python
try:
    from django.forms.utils import ErrorList
except ImportError:
    from django.forms.util import ErrorList
```

Reply:
The version comparison explains the different imports.